lms/templates/student_account/account_settings.html

```diff
@@ -51,7 +56,12 @@
         platformName,
         contactEmail,
         allowEmailChange,
-        socialPlatforms
+        socialPlatforms,
+
+        syncLearnerProfileData,
+        enterpriseName,
+        enterpriseReadonlyAccountFields,
+        edxSupportUrl
```

Comment:
can we add trailing `,` here? so next time it will not marked dirty.

Reply:
this is not python code.

Reply:
Did u check in html file?